manifests/director/postgresql.pp

```diff
@@ -40,6 +54,17 @@
     require     => [
       File[$make_bacula_tables],
       Postgresql::Server::Db[$db_name],
-    ]
+    ],
+    unless      => "/usr/bin/test -f ${make_bacula_tables_file}",
+  }
+
+  exec { "/bin/sh ${make_bacula_tables_file}":
+    user        => $user,
+    refreshonly => true,
+    subscribe   => Postgresql::Server::Db[$db_name],
+    notify      => Service[$services],
+    require     => Postgresql::Server::Db[$db_name],
+    onlyif      => "/usr/bin/test -f ${make_bacula_tables_file}",
```

Comment:
If the above `make_bacula_tables_file` variable is undef here, it might be simpler to wrap this resource in a `if $make_bacula_tables_file {...`.  What do you think?